relational_operators/CreateIndexOperator.hpp

```diff
@@ -67,7 +67,7 @@ class CreateIndexOperator : public RelationalOperator {
                       const std::vector<IndexSubBlockDescription> &index_descriptions)
       : relation_(DCHECK_NOTNULL(relation)),
         index_name_(index_name),
-        index_descriptions_(index_descriptions) {}
+        index_descriptions_(std::move(index_descriptions)) {}
```

Comment:
You should change Line 67 to `rvalue`, as well as use `move` in `ExecutionGenerator::convertCreateIndex`.